python/car.py

```diff
@@ -48,8 +48,7 @@ def match_speed_now(self,other_car):
                 other_car.get_follow_distance(safety_treshold = 0.1) <
                 other_car.distance_taken)
 
-    #HELP:
-    #should return the distance "self" takes until it reaches "speed" during deceleration/acceleration
-    #same as get_follow_distance() just for any speed
-    def match_speed_distance(speed):
+    #HELP
+    def distance_while_accelerating(current_speed, target_speed, acceleration):
```

Comment:
@torta211 Ezt is megcsinálnád kérlek Tomi? Ugyanaz mint a másik, csak két tetszőleges speed közötti gyorsulásra/lassulásra. Mindkettő jó lenne, függvény névnek nem tudom mi lenne értelmesebb, a fékezés végülis negítív gyorsulás a fizikában szóval ezt adtam :D